pkg/ragengine/controllers/preset-rag.go

```diff
@@ -20,7 +20,7 @@ import (
 
 const (
 	ProbePath = "/health"
-	Port5000  = int32(5000)
+	Port5000  = int32(8000)
```

Comment:
```suggestion
	Port8000  = int32(8000)
```

Reply:
Just wondering, is something else on 5000

Reply:
No, just workspace is on 5000